plugins/catalog-backend/src/processing/DefaultCatalogProcessingEngine.ts

```diff
@@ -116,11 +116,16 @@ export class DefaultCatalogProcessingEngine implements CatalogProcessingEngine {
             });
           }
 
+          const location =
+            unprocessedEntity?.metadata?.annotations?.[
+              'backstage.io/managed-by-location'
+            ];
+          const owner = unprocessedEntity?.spec?.owner;
```

Comment:
This one's more controversial I'd say. A bit lower value, appears in only a few kinds, and the semantics are "incomplete" so to speak anyway. We discourage reading the owner field as you know, since it might be getting its relations from any number of sources anyway. Maybe remove this one?

Reply:
My thought was that it is nice when you want to reach out to the owner if it is available. On the other hand, we can look it up manually from the location.